storm-core/src/jvm/org/apache/storm/utils/DisruptorQueue.java

```diff
@@ -63,6 +63,18 @@
     private static final String PREFIX = "disruptor-";
     private static final FlusherPool FLUSHER = new FlusherPool();
 
+    private static int getNumFlusherPoolThreads() {
+        int numThreads = 100;
+        try {
+            String threads = System.getProperty("num_flusher_pool_threads", "100");
```

Comment:
I agree that it does not add a lot of value in it's current form.  If you want me to document it I am happy to.